pkg/v1/mutate/mutate.go

```diff
@@ -161,6 +162,33 @@ func Config(base v1.Image, cfg v1.Config) (v1.Image, error) {
 	return image, nil
 }
 
+// ConfigFile mutates the provided v1.Image to have the provided v1.ConfigFile
+func ConfigFile(base v1.Image, cfg *v1.ConfigFile) (v1.Image, error) {
```

Comment:
Do we actually need this - or can we use Config() above and set the ConfigFile.Config?

Reply:
We change more than just ConfigFile.Config, so either we do this or we set the ConfigFile directly in the StripConfig method then call Config() to set everything else correctly. This felt better.